react/Viewer/ShortcutViewer.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import { ButtonLink } from 'cozy-ui/transpiled/react/Button'
+import { useI18n } from '../I18n'
+import { useClient, useFetchShortcut } from 'cozy-client'
+import get from 'lodash/get'
+import NoViewer from './NoViewer'
+import { FileDoctype } from '../proptypes'
+
+const ShortcutViewer = ({ file }) => {
+  const client = useClient()
+  const { t } = useI18n()
+  const { shortcutInfos } = useFetchShortcut(client, file.id)
+  let url = ''
+  if (shortcutInfos) {
+    url = new URL(shortcutInfos.data.attributes.url)
+  }
+  return (
+    <NoViewer
```

Comment:
Nit: A Viewer rendering NoViewer ? It's a bit surprising, is it intended ?

Reply:
Yep, I think the NoViewer should be renamed to something else, like DefaultViewer ;)